curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646c48df8674cf2b91020ecc.md

```diff
@@ -17,19 +17,12 @@ Your should have a `link` element.
 assert.match(code, /<link/)
 ```
 
-Your `link` element should have a `rel` attribute set to `stylesheet`.
+Your `link` element should have a `rel` and `href` attribute set to `stylesheet` and `./styles.css`.
 
 ```js
-assert.match(code, /<link\s+rel=("|'|`)stylesheet\1/)
+assert.match(code, /<link\s+(rel=("|'|`)stylesheet\2\s+href=("|'|`)\.\/styles\.css\3|href=("|'|`)\.\/styles\.css\4\s+rel=("|'|`)stylesheet\5)\s*\/?>/)
```

Comment:
```suggestion
assert.match(code, /<link\s+(rel=("|')stylesheet\2\s+href=("|')\.\/styles\.css\3|href=("|')\.\/styles\.css\4\s+rel=("|')stylesheet\5)\s*\/?>/)
```
Hm, backticks cannot be used to specify attributes in html https://html.spec.whatwg.org/multipage/syntax.html#attributes-2

Reply:
yeah the change is made now please have a look